proposals/jib-cli-surface.md

```diff
@@ -0,0 +1,90 @@
+# Jib CLI Surface Specification
+
+Specification for a command structures exposed to users of the jib-cli
+
+## Usage
+`jib COMMAND [OPTIONS]`
+
+## Commands
+
+The jib cli exposes the following top level commands
+```
+build        build a container
+```
+
+## Options
+
+### Required
+```
+-t  --target <image-ref>  a target image reference, using the url schema defined in jib
+                          gcr.io/project/image (default is build to registry)
+                          registry://gcr.io/project/image
+                          docker://some-image-ref
+                          tar://relative/path.tar
+                          tar:///aboslute/path.tar
+```
+
+### Optional
+
+
+#### Build Config
+```
+    --application-cache <directory>        location of the application cache
+    --base-image-cache <directory>         location of the base image cache
```

Comment:
define default?

Reply:
I think there's some work to be done here to define a better default, but the default that jib-core uses is a temporary directory.

Reply:
added to "Other Considerations" section